packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -463,7 +463,10 @@ export class Outbox {
 
 	public checkpoint() {
 		return {
-			mainBatch: this.mainBatch.checkpoint(),
+			// This type-assertion is not necessary, but without it, the generated .d.ts uses a dynamic import which doesn't
+			// resolve.
+			// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+			mainBatch: this.mainBatch.checkpoint() as IBatchCheckpoint,
```

Comment:
Would an alternate solution be:
```typescript
const mainBatch: IBatchCheckpoint = this.mainBatch.checkpoint();
return {
    mainBatch,
```
or just
```typescript
const mainBatch = this.mainBatch.checkpoint();
return {
    mainBatch,
```
(`as` is actual kind of evil and we should avoid.)

Reply:
I always forget about declaring the type on the left side as an alternative to `as`. I'll fix this to do that.

The other form is equivalent to what it was before, I think. If the type is inferred, there's a dynamic import. I think. I'll try both.

Reply:
The first form works, but the second doesn't. It still generates the dynamic import. It occurred to me that tsc-multi
_should_ be rewriting these imports, but there must be a case missing. I think this is a fine workaround, though.